mesh/nurbs.hpp

```diff
@@ -485,16 +497,43 @@ class NURBSPatchMap
 
 // Inline function implementations
 
-inline double &NURBSPatch::operator()(int i, int j)
+inline double &NURBSPatch::slice(int i, int j)
 {
    return data[j%sd + sd*(i + (j/sd)*nd)];
 }
 
-inline const double &NURBSPatch::operator()(int i, int j) const
+inline const double &NURBSPatch::slice(int i, int j) const
 {
    return data[j%sd + sd*(i + (j/sd)*nd)];
 }
 
+
+inline double &NURBSPatch::operator()(int i, int l)
+{
+#ifdef MFEM_DEBUG
+   if (data == 0 || i < 0 || i >= ni || nj > 0 || nk > 0 ||
+       l < 0 || l >= Dim)
+   {
+      mfem_error("NURBSPatch::operator() 1D");
```

Comment:
The meaning of `1D` in the error message is unclear.

Reply:
The first int gives the location of the net, the second int the variable  (coord or weight).

In the 2D and 3D cases there are also n+1 arguments.